src/Symfony/BufferedConsoleOutput.php

```diff
@@ -41,4 +43,9 @@ public function setErrorOutput(OutputInterface $error): void
     {
         $this->stderr = $error;
     }
+
+    public function section(): ConsoleSectionOutput
+    {
+        // @todo
```

Comment:
You can copy the implementation from Symfony, as I did in https://github.com/drush-ops/drush/commit/e3acd1c6c34f24535099ebc11436ab2c2a74fdd0